### ISSUE TITLE
Refactor project 1 (only as needed)

### ISSUE THREAD

**Issue by tommeagher** [ireapps/pycar#3]
Chris, I see you've made some commits on this already. Holler if you need any help.

**Comment by tommeagher** [comment]
Chris, how's this coming along? How do you feel about it?

**Comment by chrislkeller** [comment]
Hey Tom.. I feel good. I hope to have a final draft by the end of the weekend for feedback and guidance.

**Comment by tommeagher** [comment]
Awesome. Look forward to checking it out. Thanks!

**Comment by chrislkeller** [comment]
Let me revise my previous statement to say "draft" sted "final draft." I have some blanks to fill in, but I think the re-focus is basically completed. 

I took the feedback you posted to heart, and thought through what I stumbled against in teaching and tried to just focus on the building blocks.

I have some work to do building out sections on comments, indentation and comparisons and conditionals but the gist of it is there, and the "show you how to do a practical task" portion is complete and read for feedback.

I'll open a pull request later this week as I polish, but wanted to show some progress.
